### ISSUE TITLE
progress bar prints multiple lines, with Non-ASCII Prefix / Postfix

### ISSUE THREAD

**Issue by kakakaya** [cheggaaa/pb#90]
I used pb's progress bar with unicode character's prefix, and got multiple lines of output.

my code is below...
```go
package main

import (
	"gopkg.in/cheggaaa/pb.v1"
	"time"
)

func main() {
	count := 100000
	// header := "progress" // This works well
	header := "進捗" // "progress" in Japanese, this cause ugly

	bar := pb.StartNew(count).Prefix(header)
	for i := 0; i < count; i++ {
		bar.Increment()
		time.Sleep(time.Millisecond)
	}
	bar.FinishPrint("The End!")
}
```

and I got output as below.
![2016-11-21 16_41_52-miracle mysql 5 5 rakumovis - oracle vm virtualbox](https://cloud.githubusercontent.com/assets/3298037/20474387/81fa2d64-b009-11e6-9c20-84793f5e50b7.png)

**Comment by kakakaya** [comment]
And, if the prefix is long enough, it pushes out percent and time remaining.
```golang
package main

import (
	"gopkg.in/cheggaaa/pb.v1"
	"time"
)

func main() {
	count := 100000
	// header := "progress" // This works well
	// header := "進捗"         // "progress" in Japanese, this cause ugly
	header := "進捗進捗進捗進捗進捗" // "progressprogressprogressprogressprogress" in Japanese, this cause SUPER UGLY

	bar := pb.StartNew(count).Prefix(header)
	for i := 0; i < count; i++ {
		bar.Increment()
		time.Sleep(time.Millisecond)
	}
	bar.FinishPrint("The End!")
}
```
![2016-11-21 16_51_46-miracle mysql 5 5 rakumovis - oracle vm virtualbox](https://cloud.githubusercontent.com/assets/3298037/20474698/2c84420a-b00b-11e6-93d3-58707b7b67ba.png)

**Comment by cheggaaa** [comment]
Fixed and v1.0.7 released

**Comment by kakakaya** [comment]
Thanks!
